src/main/java/com/minecolonies/coremod/colony/CitizenData.java

```diff
@@ -1623,6 +1617,8 @@ public void onGoSleep()
               Component.translatable(CITIZEN_NOT_GUARD_NEAR_HOME),
               ChatPriority.CHITCHAT));
         }
+
+        decreaseSaturation(workBuilding == null || workBuilding.getBuildingLevel() == 0 ? 1 : (SATURATION_DECREASE_FACTOR * Math.pow(2, workBuilding.getBuildingLevel())) * 2);
```

Comment:
where did this come from?

Reply:
from citizens desiredactivity, was triggered when it decided to go to sleep

Reply:
Ah ok